modules/search/instant-search/components/gridicon/index.jsx

```diff
@@ -324,6 +325,12 @@ class Gridicon extends Component {
 						<path d="M21 19l-5.154-5.154C16.574 12.742 17 11.42 17 10c0-3.866-3.134-7-7-7s-7 3.134-7 7 3.134 7 7 7c1.42 0 2.742-.426 3.846-1.154L19 21l2-2zM5 10c0-2.757 2.243-5 5-5s5 2.243 5 5-2.243 5-5 5-5-2.243-5-5z" />
 					</g>
 				);
+			case 'gridicons-jetpack-search':
```

Comment:
I spent a while trying to get this to display well, but kept running into theme problems. So decided to punt on it, but left the code in for now.